include/color_spinor_field_order.h

```diff
@@ -1184,31 +1201,39 @@ namespace quda
       using AllocInt = typename AllocType<huge_alloc>::type;
       using norm_type = float;
       Float *field = nullptr;
+#if 0
```

Comment:
Marking this just so it doesn't get lost---I'm guessing this can reduce the size of the accessor, with the trade-off being extra runtime work?

Reply:
Exactly, this reduces the size of the accessor, allowing us to fit 16 RHS without large args.  I tested the performance with and without this: conclusion was that at both extremes (`n_rhs=1` and `n_rhs = 16`) performance was identical.  This basically swaps some LDCs for a bit of math (which is hoisted out by the compiler and only performed once).  Doesn't change the register count, and the new local `norm` is plopped in a uniform register.

Reply:
I was thinking to leave the code in there, in case for some reason it ever needs to come back in the future.

I'll mark this as resolved.